quiz_app/quiz.sql

```diff
@@ -1,8 +1,8 @@
 CREATE TABLE quiz_app
```

Comment:
Nice. A good next step might be to add a table for `attempts` or something like that to capture answers people provided. At that point, you could check to see how many answered correctly, etc.